service/api/pokemon.go

```diff
@@ -0,0 +1,67 @@
+package api
+
+import (
+	"bootcamp/config"
+	"bootcamp/domain/model"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+type pokeApiServiceInterface interface {
+	fetchPokeApi(name string) (model.FetchedPokemon, error)
+}
+
+type PokeApiService struct{}
+
+func (s PokeApiService) fetchPokeApi(name string) (model.FetchedPokemon, error) {
+	var pokemon model.FetchedPokemon
+	configuration := config.GetConfiguration()
+	pokeApiUrl := configuration.PokeApiUrl + "/" + name
+
+	client := http.Client{
+		Timeout: time.Second * 5,
+	}
+
+	req, err := http.NewRequest(http.MethodGet, pokeApiUrl, nil)
+
+	if err != nil {
```

Comment:
Make sure you're also logging the error you actually got, this is helpful for debugging purposes

Reply:
👌